macrosynergy/pnl/contract_signals.py

```diff
@@ -280,6 +351,7 @@ def contract_signals(
     hratios: Optional[str] = None,
     start: Optional[str] = None,
     end: Optional[str] = None,
+    est_freq: str = "M",
```

Comment:
Why does contract signals have an estimation frequency? Shouldn't it be a `rebalance_freq` if any?

Reply:
Was uncertain which one this was. Will change it out soon